PWG/EMCAL/EMCALbase/AliEMCALRecoUtils.h

```diff
@@ -599,6 +620,11 @@ class AliEMCALRecoUtils : public AliEMCALRecoUtilsBase {
   TArrayL64  fGlobalEventID;                   ///< Global event ID
   Bool_t     fDoUseMergedBC;                   ///< flag for using one histo for all BCs
 
+  // energy dependent time clalibration
+  Bool_t     fTimeECorrection;            ///< Switch on or off the energy dependent time recalibration
+  TSpline3*  fEMCALTimeEShiftCorrection;  ///< Spline to correct energy dependent time shift for high gain cells
```

Comment:
I didn't look closely at the RecoUtils, but I don't think you want this to stream, right? (ie. should be `//!<!`

Reply:
would prefer to leave this as is, as its done in the same manner for BC map and all other corrections